### PR TITLE
Added files for the Infinity Mini

### PR DESCRIPTION
**Pull request by unknown-loser**
Hi there, 

RoboTech 3D is a small business that manufactures custom 3D printers and filament

We have recently released our second production model the Infinity Mini and have 2 more models in R&D that will be released in the coming months

This pull request adds all the necessary files to make a machine Profile for the Infinity Mini :D

Signed-off-by: unknown-loser <unknown_loser.exe@protonmail.com> <github@robotech3d.co.za>

### CODE REVIEW

**Comment by nallath** [comment]
There seem to be some issues with this. Could you fix the following issues:

 E       AssertionError: Unnecessary default_values for ['cool_fan_speed', 'infill_pattern'] in /__w/Cura/Cura/tests/Settings/../../resources/definitions/robotech3d_infinity_base.def.json
E       assert not {'cool_fan_speed', 'infill_pattern'}

**Comment by unknown-loser** [comment]
> There seem to be some issues with this. Could you fix the following issues:
> 
> E AssertionError: Unnecessary default_values for ['cool_fan_speed', 'infill_pattern'] in /__w/Cura/Cura/tests/Settings/../../resources/definitions/robotech3d_infinity_base.def.json E assert not {'cool_fan_speed', 'infill_pattern'}

I will fix it asap

**Comment by unknown-loser** [comment]
@nallath I have fixed the issue with regards to that error message

File: resources/definitions/robotech3d_infinity_base.def.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "acceleration_print":                           { "default_value": 2200 },
+        "acceleration_wall_0":                          { "value": 600 },
+        "acceleration_layer_0":                         { "value": 600 },
+        "acceleration_travel_layer_0":                  { "value": 600 },
+        "acceleration_roofing":                         { "value": 600 },
+        "jerk_enabled":                                 { "default_value": false },
+        "jerk_wall_0":                                  { "value": 10 },
+        "jerk_roofing":                                 { "value": 10 },
+        "cool_fan_speed":                               {"default_value": 25},
+        "infill_sparse_density":                        {"default_value": 10},
```
Comment (nallath) [inline resources/definitions/robotech3d_infinity_base.def.json:144]:
```suggestion
        "infill_sparse_density":                        {"default_value": 10}
```

Trailing , is not allowed in JSON. Please test the files in Cura first before asking for a review; This file won't load in Cura as it is now.

Comment (unknown-loser) [reply resources/definitions/robotech3d_infinity_base.def.json]: My mistake, I forgot to check for trailing comma's after editing it to fix the previous error

**Comment by unknown-loser** [comment]
I'll resubmit later

**Pull request closed without merging**